tests/tracking/_model_registry/test_utils.py

```diff
@@ -19,14 +19,39 @@
 
 
 def test_set_get_registry_uri():
-    uri = "databricks://registry/path"
-    set_registry_uri(uri)
-    assert get_registry_uri() == uri
+    with mock.patch("mlflow.tracking._model_registry.utils.get_tracking_uri") \
+            as get_tracking_uri_mock:
+        get_tracking_uri_mock.return_value = "databricks://tracking_sldkfj"
+        uri = "databricks://registry/path"
+        set_registry_uri(uri)
+        assert get_registry_uri() == uri
+        set_registry_uri(None)
 
 
 def test_set_get_empty_registry_uri():
-    set_registry_uri("")
-    assert get_registry_uri() == ""
+    with mock.patch("mlflow.tracking._model_registry.utils.get_tracking_uri") \
+            as get_tracking_uri_mock:
+        get_tracking_uri_mock.return_value = None
+        set_registry_uri("")
+        assert get_registry_uri() is None
+        set_registry_uri(None)
+
+
+def test_default_get_registry_uri_no_tracking_uri():
+    with mock.patch("mlflow.tracking._model_registry.utils.get_tracking_uri") \
+            as get_tracking_uri_mock:
+        get_tracking_uri_mock.return_value = None
+        set_registry_uri(None)
+        assert get_registry_uri() is None
+
+
+def test_default_get_registry_uri_with_tracking_uri_set():
+    tracking_uri = "databricks://tracking_werohoz"
+    with mock.patch("mlflow.tracking._model_registry.utils.get_tracking_uri") \
+            as get_tracking_uri_mock:
+        get_tracking_uri_mock.return_value = tracking_uri
+        set_registry_uri(None)
+        assert get_registry_uri() == tracking_uri
 
```

Comment:
did not add tests for `_resolve_registry_uri` as it's fairly well-covered by the client tests